pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -449,18 +461,20 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 			// same way it handles the "istio-ca-secret" secret. Isitod utilizes a secret watch instead
 			// of file watch to check for secret updates. This may change in the future, and istiod
 			// will watch the file mount instead.
+
+			// TODO(costin): we can use the presence of intermediary certs to know if we can rotate.
 		}
 
 		// Either the secret is not mounted because it is named `istio-ca-secret`,
 		// or it is `cacerts` secret mounted with "istio-generated" key set.
-		caOpts, err = s.createSelfSignedCACertificateOptions(&fileBundle, opts)
+		caOpts, err = s.loadCAFromSecrets(&fileBundle, opts)
 		if err != nil {
 			return nil, err
 		}
 		caOpts.OnRootCertUpdate = s.updateRootCertAndGenKeyCert
 	} else {
 		// The secret is mounted and the "istio-generated" key is not used.
-		log.Info("Use local CA certificate")
+		log.Info("Use local CA certificate from /etc/cacert, not auto-generated by Istiod")
```

Comment:
To be more correct, `/etc/cacert` can be customized

Reply:
Right. Do you happen to know how :-) ? Happy to update the comment with the actual source of truth - the goal is to allow user to be very clear on where their source of trust is coming from...

Reply:
How can the mount path be customized?

Reply:
I see a  env `ROOT_CA_DIR`,  istio can be deployed in VM in theory. Though i never know anyone do that way